packages/wallets/polkadot.js/src/polkadot.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  Chain,
+  type ConnectWalletParams,
+  WalletOption,
+  setRequestClientConfig,
+} from "@swapkit/helpers";
+import { getWalletForChain } from "./helpers";
+
+const POLKADOT_SUPPORTED_CHAINS = [Chain.Polkadot] as const;
```

Comment:
I *think* we can add `Chain.FLIP` here as its a substrate based chain

Reply:
Done